src/Core/TTC.idr

```diff
@@ -749,6 +749,7 @@ TTC CDef where
 export
 TTC CG where
   toBuf b Chez = tag 0
+  toBuf b ChezSep = tag 1
```

Comment:
You should bump the version number in `Core.Binary`

Reply:
Thanks! Done.